src/test/java/com/redhat/lightblue/camel/utils/LightblueInsertRequestTransformer.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.lightblue.camel.transformer;
+package com.redhat.lightblue.camel.utils;
```

Comment:
@paterczm while save might be enough for current entities we are dealing with, that will soon not be enough for whats coming up. infact I have an investigation task right now to ensure save will work for the current needs too. we cannot assume all information will be present in the incoming message.

also, since this is a generic lightblue camel utility I think it makes sense to have a generic transformer. not sure what LiteralDataRequest is, will wait for that.